roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -65,16 +80,13 @@ def run(self, tmp, task_vars):
 
     def required_images(self, task_vars):
         deployment_type = get_var(task_vars, "deployment_type")
-        # FIXME(juanvallejo): we should handle gracefully with a proper error
-        # message when given an unexpected value for `deployment_type`.
-        image_base_name = self.docker_image_base[deployment_type]
+        image_base_name = self.docker_image_base.get(deployment_type)
+        if not image_base_name:
+            msg = "Unable to determine required images for deployment type {deployment}"
+            return set(), msg.format(deployment=deployment_type)
```

Comment:
Hmm, this function sometimes return 2 values, sometimes 1.

I think in this case we can probably use the `OpenShiftCheckException` to simplify control flow.

```python
if deployment_type not in self.docker_image_base:
    raise OpenShiftCheckException(...msg here...)
image_base_name = self.docker_image_base[deployment_type]
...
```